### PR TITLE
Ignore keyPrefix property if namespace prefix used

### PR DESCRIPTION
**Pull request by lexxor**
Hi!

keyPrefix property should be ignored if key with namespace prefix used. Something like `$t('common:cancel')`

### CODE REVIEW

**Comment by claudiocro** [comment]
Thank you very much!

In this project we use semantic-release. Can you rephrase your commit message to `fix: Ignore keyPrefix property if namespace prefix used`?

FYI: [commit message format](https://github.com/semantic-release/semantic-release#default-commit-message-format)

Thx!

**Comment by lexxor** [comment]
@claudiocro done
